src/functions/content-types/product/item.ts

```diff
@@ -80,3 +82,19 @@ function normalizeToCM(volume, unit) {
       throw new Error(`Unsupported volume unit: ${unit}`);
   }
 }
+
+function calculateNeedsMetFields(data) {
+  const { items, months, people } = data;
+  let monthlyNeedsMetPerItem = (people * months) / items;
+  monthlyNeedsMetPerItem = parseFloat(monthlyNeedsMetPerItem.toFixed(2));
+  return { ...data, monthlyNeedsMetPerItem };
+}
+
+function calculateValueFields(data) {
+  // Note there is no normalizing of currency units as
+  // currently USD is the only unit available.
+  const { packagePrice, countPerPackage } = data;
+  let pricePerItemUSD = packagePrice / countPerPackage;
+  pricePerItemUSD = parseFloat(pricePerItemUSD.toFixed(2));
```

Comment:
* **note:** the 2 decimal points of precision makes more sense here since we're dealing with $, but same question about needing the `parseFloat` wrapping it

Reply:
The `parseFloat` keeps `pricePerItemUSD ` a number after applying `toFixed()`, which converts it to a string. Let me know if there's a better way to do this. Or if we don't want the limit here.

Reply:
@madcalf you're totally right!  and in this case I agree that we should have the limit here as we're dealing with currencies.  It doesn't make sense for an item to have a value in "half cents", etc.

**Question(non-blocking):** would it make more sense to round to the nearest cent, instead of simply dropping the remainder?

Reply:
Ooops looks like `toFixed` does round!  Answered my own question lol.